docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+name: whatsync
+services:
+  # server:
+  #   build:
+  #     context: .
+  #     dockerfile: server/Dockerfile
+  #   container_name: whatsync-server
+  #   env_file:
+  #     - server/.env
+  #   environment:
+  #     - RUNNING_IN_DOCKER=true
+  #   hostname: whatsync-server
+  #   ports:
+  #     - 8080:8080
+  #   restart: unless-stopped
+
+  # web:
+  #   build:
+  #     context: .
+  #     dockerfile: web/Dockerfile
+  #   container_name: whatsync-web
+  #   depends_on:
+  #     - server
+  #   env_file:
+  #     - web/.env
+  #   environment:
+  #     - ORIGIN=http://localhost:8080
+  #     - RUNNING_IN_DOCKER=true
+  #   hostname: whatsync-web
+  #   ports:
+  #     - 80:80
+  #   restart: unless-stopped
```

Comment:
Running the 2 containers is the better way to run a server, let's have this part enable, and the single container image option commented out :)

Reply:
Those individual docker files are not working as expected, thus I had it commented out.
Additionally, the GitHub workflow is creating container image from the root `Dockerfile`, not the individual ones.

So my counter argument would be to remove the individual ones along with individual docker files and just use the root `DockerFile` & `docker-compose.yml`

Reply:
Counter argument is indeed correct, but lets keep the separate `Dockerfile`s.